<ui version="4.0" >
 <class>NewSetDialog</class>
 <widget class="QDialog" name="NewSetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Set Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtons" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>350</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="_nameText" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_menuNameText" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="groupsLabel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>120</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Groups</string>
   </property>
  </widget>
  <widget class="QPushButton" name="groupsAddButton" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>300</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>40</y>
     <width>53</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="menuNameLabel" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>80</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Menu Name</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="_groupsList" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>140</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text" >
     <string>1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="timesetsRemoveButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>500</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="groupsRemoveButton" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>300</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setsEditButton" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>190</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setsEditButton_2" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>200</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setsEditButton_3" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>190</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setsEditButton_4" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>200</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setsEditButton_5" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>190</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="groupsEditButtons" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>300</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>accepted()</signal>
   <receiver>NewSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>rejected()</signal>
   <receiver>NewSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
